package.json

```
@@ -42,37 +42,37 @@
   ],
   "license": "GPL-3.0",
   "devDependencies": {
-    "@jest-runner/electron": "^3.0.0",
-    "electron": "^11.1.1",
+    "@jest-runner/electron": "^3.0.1",
+    "electron": "^11.4.6",
     "electron-mocha": "^10.0.0",
     "electron-packager": "^15.2.0",
-    "electron-reloader": "^1.1.0",
-    "eslint": "^7.17.0",
+    "electron-reloader": "^1.2.1",
+    "eslint": "^7.26.0",
     "jest": "^26.6.3",
-    "mocha": "^8.2.1",
+    "mocha": "^8.4.0",
     "npm-run-all": "^4.1.5",
     "nyc": "^15.1.0",
-    "patch-package": "^6.2.2",
+    "patch-package": "^6.4.7",
     "pre-commit": "^1.2.2",
     "spectron": "^13.0.0",
-    "stylelint": "^13.8.0",
+    "stylelint": "^13.13.1",
     "stylelint-config-standard": "^20.0.0",
     "tap-spec": "^5.0.0",
-    "tape": "^5.1.1"
+    "tape": "^5.2.2"
   },
   "dependencies": {
-    "@fortawesome/fontawesome-free": "^5.15.1",
-    "bootstrap": "^4.5.3",
-    "date-holidays": "^1.8.5",
+    "@fortawesome/fontawesome-free": "^5.15.3",
+    "bootstrap": "^4.6.0",
```

```suggestion
    "bootstrap": "^5.0.0",
```

Latest is v5, did you not upgrade due to some problem? I'd expect this one to be hassle-free

---

This is the upgrade done automatically by npm, and because we specify with `^` it will upgrade only minor and patches, not major versions. In this case I don't see a reason why not go to 5, so let's try...